components/Reviews/Stats.js

```diff
@@ -0,0 +1,116 @@
+import { useState, useEffect } from "react";
+import StarRating from "../StarRating";
+import { IoIosArrowDown, IoIosArrowUp } from "react-icons/io";
+
+import style from "../../styles/RevStats.module.css";
+const Stats = () => {
+  const [totalRatings] = useState(23);
+  const [ratingDistribution] = useState([
+    { star: 5, count: 18 },
+    { star: 4, count: 3 },
+    { star: 3, count: 2 },
+    { star: 2, count: 0 },
+    { star: 1, count: 0 },
+  ]);
+  const [rating, setRating] = useState(0);
+
+  const findRating = () =>
+    setRating(
+      ratingDistribution.reduce((prev, current, i) => {
+        let prior = i === 1 ? prev.count * prev.star : prev;
+        let present = current.count * current.star;
+        return i === ratingDistribution.length - 1
+          ? Math.round(((prior + present) / totalRatings) * 10) / 10
+          : prior + present;
+      })
+    );
```

Comment:
get average rating, will update in future to only count verified purchases if time permits